src/connectors/bandcamp.js

```diff
@@ -25,7 +25,7 @@ function getArtist() {
 	if (artist === null) {
 		artist = $('.waypoint-artist-title').text().substr(3);
 	}
-	return artist;
+	return $.trim(artist);
```

Comment:
Why did you add trim? It's useless because trim is done in the default metadata filter.

Reply:
I observed different behaviour

Reply:
```
Web Scrobbler: {
  "track": "Proper Preaching",
  "artist": "Luca Musto",
  "album": "Proper Preaching EP",
  "albumArtist": " \n          Luca Musto\n          ",
  "uniqueID": null,
  "duration": null,
  "currentTime": null,
  "isPlaying": true,
  "trackArt": "https://f4.bcbits.com/img/a1481550943_16.jpg"
}
```

Reply:
What do you mean? If you see **artist** field is not trimmed, it should be fixed in another place.

Reply:
I noticed the `isArtistLetious` function is broken because of raw (i.e. non-trimmed) artist passed as a function argument. If it's a reason why you added trim call, I suggest to fix `isArtistLetious` function instead.

Not sure, if we should do it in this PR, but maybe in #2006 or another.